docs/docs/modules.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: KAMI Blue Modules
+description: KAMI Blue Module List
+---
+
+## KAMI Blue Modules
+
+<details>
+    <summary>Capes</summary>
```

Comment:
These aren't modules?

Reply:
Then why is it in the `modules` folder in kamiblue?

Reply:
They are not registered modules